chef/cookbooks/crowbar/templates/default/crowbar-router-migration.sh.erb

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# This script migrates all neutron routers to the node where the script is
+# executed. As part of the upgrade, the founder node disbanded all routers
+# to other nodes. After the founder is upgraded, all routers have to come
+# back.
+
+LOGFILE=/var/log/crowbar/node-upgrade.log
+UPGRADEDIR=/var/lib/crowbar/upgrade
+mkdir -p "`dirname "$LOGFILE"`"
+exec >>"$LOGFILE" 2>&1
+
+echo "Executing $BASH_SOURCE"
+
+set -x
+
+mkdir -p $UPGRADEDIR
+
+if [[ -f $UPGRADEDIR/router-migration-ok ]] ; then
+    echo "router migration actions were already successfully executed"
+    exit 0
+fi
+
+source /root/.openrc
```

Comment:
We need to suppress logging around this, so we do not log the password. See https://github.com/crowbar/crowbar-core/blob/master/chef/cookbooks/crowbar/templates/default/crowbar-pre-upgrade.sh.erb#L26